webapp/src/ts/modules/messages/messages.component.html

```diff
@@ -0,0 +1,40 @@
+<div class="filters">
+    <div class="inner">
+        <!--<mm-navigation></mm-navigation>-->
+    </div>
+</div>
+
+<div class="inner">
+    <div class="inbox page">
+
+        <div id="message-list" class="col-sm-4 inbox-items left-pane">
+            <ul *ngIf="!loading">
+                <li class="content-row"
+                    *ngFor="let msg of conversations; trackBy: listTrackBy"
+                    [ngClass]="{
+                        'selected': msg.key === loadingContent || (!loadingContent && selectedConversation && msg.key === selectedConversation.id),
+                        'unread': !msg.read
+                    }">
+                    <a [routerLink]="['messages', msg.type, msg.key]">
+                        <div class="content">
+                            <div class="heading">
+                                <h4>{{msg.contact || msg.from || ('messages.unknown.sender' | translate)}}</h4>
+                                <div class="date"><span [innerHTML]="msg.date | relativeDate"></span></div>
+                            </div>
+                            <div class="summary"><p>{{ msg.message }}</p></div>
+                            <div class="detail" *ngIf="msg.lineage && msg.lineage.length" [innerHTML]="msg.lineage | lineage"></div>
+                        </div>
+                    </a>
+                </li>
+            </ul>
+
+            <p *ngIf="error" class="alert alert-danger" role="alert" translate>Error fetching messages</p>
+            <p class="loading-status" *ngIf="!error && !loading && !conversations.length" translate>No messages found</p>
+            <p class="loading-status" *ngIf="!error && !loading && conversations.length">{{ 'message.list.complete' | translate }}</p>
+            <div class="loader" *ngIf="loading"></div>
+            <div class="padding"></div>
+        </div>
+
+        <router-outlet></router-outlet>
+    </div>
+</div>
```

Comment:
Indentation is all off - make sure your editor is configured correctly.

Reply:
Fixed